docs/networking/connecting-the-arena-to-a-network.md

```diff
@@ -58,12 +59,12 @@ To change the IP address of the TNS in the console:
 Power and UPS behaviour 
 -----------------------
 
-The TitanNet switch in the console runs from the same UPS as the console
+The network switch in the console runs from the same UPS as the console
 itself, so if power fails the switch will continue working.
 
-When you power down the Arena, the UPS will continue to power the
+When you power down the console, the UPS will continue to power the
 internal switch for about 5 minutes. This is to allow you to restart the
```

Comment:
```suggestion
internal switch for about 5 minutes or until the external power is subsequently removed. This is to allow you to restart the
```

Reply:
OK let me check I have that clear, so if you pull the power while the console is on then the switch will go over to UPS, but once you have shut down the console then the switch will go off if you pull the power?

Reply:
That is my understanding yes (I just checked this with JB)